src/main/java/org/activiti/cloud/examples/controllers/ConnectorController.java

```diff
@@ -18,4 +20,9 @@ public String welcome() {
                 "  \"var1\" : \""+exampleConnector.getVar1Copy()+"\" }";
     }
 
+    @RequestMapping(method = RequestMethod.GET, path = "/v1/home-inbound-variables")
+    public String welcomeInputVariables(){
```

Comment:
Then here you have:

```
    public Map<String, Object> welcomeInputVariables(){
        return exampleConnector.getInBoundVariables();
    }
```

Reply:
I thought about it, but then i wanted to have just a string to make the response simpler.. but I could do it I think